rust-runtime/smithy-http/src/property_bag.rs

```diff
@@ -176,6 +174,34 @@ impl fmt::Debug for PropertyBag {
     }
 }
 
+/// A new-type of [`PropertyBag`] that can be safely shared across threads and cheaply cloned.
```

Comment:
A "wrapper type"? this isn't really a newtype because a newtype is a direct tuple struct with no container.